src/sts.rs

```diff
@@ -0,0 +1,278 @@
+//! The AWS STS API.
+//!
+//! If you're using the service, you're probably looking for [StsClient](struct.StsClient.html).
+#![allow(unused_imports)]
```

Comment:
I'd like to remove this so we only import what we need.  When I remove the line there's four unused imports.  I'll look at it some more to see what it takes to remove this and still have the `sts` feature compile and work.